frontend/packages/dev-console/integration-tests/support/step-definitions/common/common.ts

```diff
@@ -35,7 +35,7 @@ Given('user has created namespace starts with {string}', (projectName: string) =
 
 Given('user has created or selected namespace {string}', (projectName: string) => {
   Cypress.env('NAMESPACE', projectName);
-  projectNameSpace.selectOrCreateProject(`${projectName}`);
+  projectNameSpace.selectOrCreateProjectForVersion49(`${projectName}`);
```

Comment:
There seems to be no reason to change a step common for all the tests. I've tried running the tests with the original `selectOrCreateProject` and they seem to be running just as well.

Reply:
Strange... Let me recheck it again.

Reply:
Maybe I  something lost but when I Opened the Widget: 
![Selection_024](https://user-images.githubusercontent.com/1640058/142207419-62721702-e19a-44c4-ac08-a73c5bf41f4e.png)
I did not find the WebElement like: https://github.com/openshift/console/blob/master/frontend/packages/dev-console/integration-tests/support/pages/app.ts#L174

Reply:
Looks like it works as expected under the admin perspective but if we login under regular user it fails, we need to modify this function for both cases.